### PR TITLE
Introduce fitted format

### PR DESCRIPTION
**Pull request by lukemelia**

### CODE REVIEW

File: packages/base/default-templates/embedded.gts [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+          class='card-description'
+          data-test-card-description
+        >{{@model.description}}</div>
+      {{else}}
+        {{! empty links-to field }}
+        <div data-test-empty-field class='empty-field'></div>
+      {{/if}}
+    </div>
+    <style>
+      .fitted-template {
```
Comment (tintinthong) [inline packages/base/default-templates/embedded.gts]: wrong class naming?

File: packages/base/default-templates/embedded.gts [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+        font: 500 var(--boxel-font-xs);
+        line-height: 1.27;
+        letter-spacing: 0.11px;
+        overflow: hidden;
+      }
+      .thumbnail-section {
+        justify-content: center;
+      }
+
+      @container fitted-card (width < 150px) {
```
Comment (tintinthong) [inline packages/base/default-templates/embedded.gts:97]: fitted-card

File: packages/base/default-templates/embedded.gts (hunk 1 above)
Comment (lukemelia) [reply packages/base/default-templates/embedded.gts]: Oy, good catch

File: packages/base/default-templates/embedded.gts [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+        .card-title {
+          font: 500 var(--boxel-font-sm);
+          line-height: 1.23;
+          letter-spacing: 0.13px;
+        }
+      }
+
+      @container fitted-card (width < 200px) {
```
Comment (tintinthong) [inline packages/base/default-templates/embedded.gts:105]: fitted-card?

File: packages/experiments-realm/Product/4c2a356e-ec74-4a49-9aa2-a88df64cf446.json [hunk 4]
```diff
@@ -31,12 +31,12 @@
       },
       "unitPrice.currency": {
         "links": {
-          "self": "../Currency/2"
+          "self": "http://localhost:4201/experiments/Currency/2"
```
Comment (habdelra) [inline packages/experiments-realm/Product/4c2a356e-ec74-4a49-9aa2-a88df64cf446.json]: don't use absolute URL's. probably this is the result of a bug, do you recall how this was altered?

File: packages/experiments-realm/Person/1.json [hunk 5]
```diff
@@ -26,12 +26,12 @@
       },
       "trips.countriesVisited.0": {
         "links": {
-          "self": "../Country/1"
+          "self": "http://localhost:4201/experiments/Country/1"
```
Comment (habdelra) [inline packages/experiments-realm/Person/1.json]: ditto on the absolute URL

File: packages/experiments-realm/Person/1.json [hunk 6]
```diff
@@ -26,12 +26,12 @@
...
         }
       },
       "trips.countriesVisited.1": {
         "links": {
-          "self": "../Country/2"
+          "self": "http://localhost:4201/experiments/Country/2"
```
Comment (habdelra) [inline packages/experiments-realm/Person/1.json]: absolute URL

File: packages/experiments-realm/Product/4c2a356e-ec74-4a49-9aa2-a88df64cf446.json [hunk 7]
```diff
@@ -31,12 +31,12 @@
...
         }
       },
       "shippingCost.currency": {
         "links": {
-          "self": "../Currency/2"
+          "self": "http://localhost:4201/experiments/Currency/2"
```
Comment (habdelra) [inline packages/experiments-realm/Product/4c2a356e-ec74-4a49-9aa2-a88df64cf446.json]: absolute URL

**Review by habdelra — APPROVED** [review]
looks good, just some issues with absolute URL's in the instance data

File: packages/experiments-realm/Product/4c2a356e-ec74-4a49-9aa2-a88df64cf446.json (hunk 4 above)
Comment (lukemelia) [reply packages/experiments-realm/Product/4c2a356e-ec74-4a49-9aa2-a88df64cf446.json]: No, but I suspect this: https://github.com/cardstack/boxel/blob/969a188918ffbf5aeaf083c5932cafee196d47cc/packages/experiments-realm/monetary-amount.gts#L17

**Pull request merged**